src/components/InputItem/InputItem.module.css

```diff
@@ -0,0 +1,18 @@
+.field input:focus {
+  outline: none;
+}
+
+
+@media (max-width: 1080px) {
+  .field {
+    width: 70%;
+    margin: 0 auto;
+  }
+}
+
+@media (max-width: 780px) {
+  .field {
+    width: 50%;
+    margin: 0 auto;
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)